code/modules/mob/living/living.dm

```diff
@@ -521,10 +524,10 @@
 	ExtinguishMob()
 	fire_stacks = 0
 	confused = 0
-	update_mobility()
 	GET_COMPONENT(mood, /datum/component/mood)
 	if (mood)
 		mood.remove_temp_moods(admin_revive)
+	remove_CC()
```

Comment:
```suggestion
```

Reply:
Moved to the "healing area".

Reply:
will do, needs a bit of a further change to make work